### PR TITLE
Missing translations for Chinese (zh_CN) #38732

### LINKED ISSUE #38732: Missing translations for Chinese (zh_CN)
Hello,

There are some missing translation for Chinese (zh_CN). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.zh_CN.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.zh_CN.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by fd6130**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no
| Tickets       | Fix #38732 
| License       | MIT

Added missing translations for zh_CN in Form, Security, Validator.

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.zh_CN.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter an integer.</source>
+                <target>请输入整数。</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>请选择有效的语言。</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>请选择有效的地区。</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.zh_CN.xlf]: Is this correct? Does it refer to a locale? ie, "en", "fr", sv" etc?

File: src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf [hunk 2]
```diff
@@ -66,6 +66,14 @@
                 <source>Account is locked.</source>
                 <target>帐号已被锁定。</target>
             </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>登入失败的次数过多，请稍后再试。</target>
+            </trans-unit>
+            <trans-unit id="18">
+                <source>Invalid or expired login link.</source>
+                <target>登入链接已失效，或逾期。</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf]: Can you double check if this is correct?

**Review by Nyholm — COMMENTED** [review]
Thank you. It looks good. 

I used Google Translate to confirm this. I might be very wrong in be comments below, but can you double check?

File: src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf (hunk 2 above)
Comment (fd6130) [reply src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf]: Hi, yes this is correct.

File: src/Symfony/Component/Form/Resources/translations/validators.zh_CN.xlf (hunk 1 above)
Comment (fd6130) [reply src/Symfony/Component/Form/Resources/translations/validators.zh_CN.xlf]: Hi, i will change it to "请选择有效的语言环境”

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.zh_CN.xlf]: Excellent. Thank you

File: src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf (hunk 2 above)
Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf]: Okey cool. 

Google translated this to "The login link has expired or expired." =)

Comment (fd6130) [reply src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf]: Can't really trust google translate much but i will try tweak it to nearest meaning.

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf]: Do whatever sounds correct. 

Google translate is far from perfect, but sometimes it helps me to spot some misstakes. 

If you are happy with the current one then so am I.

Comment (fd6130) [reply src/Symfony/Component/Security/Core/Resources/translations/security.zh_CN.xlf]: I change it to "失效或过期的登入链接".  Now it have the same meaning as the english one from translate.

**Review by Nyholm — APPROVED** [review]
Cool cool. 

Thank you

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fd6130.

**Pull request merged**